interfaces/mount/entry.go

```diff
@@ -103,3 +119,28 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	fields := strings.Fields(s)
```

Comment:
The glibc mntent_r.c is using `strsep(..., " \t")` only, strings.Fields() is using any unicode whitespace. I don't think it makes a difference in practise but I would prefer if we would follow glibc as close as possible.